__tests__/server-get.test.js

```diff
@@ -56,7 +56,8 @@ describe("GET/api/reviews/:review_id", () => {
           created_at: "2021-01-18T10:00:20.514Z",
           votes: 1,
           review_img_url:
-            "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png"
+            "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+          comment_count: "0"
```

Comment:
comment_count should be a number, not a string. Have a look at how to cast a piece of data into another data type in an SQL query